src/addlist.js

```diff
@@ -0,0 +1,20 @@
+import { saveStorage, getStorage } from './storage.js';
+import populateList from './populateList.js';
+
+const addNewTask = (input) => {
+  const taskList = getStorage();
+  const task = {
+    index: taskList.length + 1,
+    completed: false,
+    description: input.value,
+  };
+
+  taskList.push(task);
+
+  saveStorage(taskList);
+  input.value = '';
+  populateList(taskList);
+  return task;
+};
```

Comment:
- [ ] Kindly fix this issue where on adding a task, an empty task is created too as seen below

![image](https://user-images.githubusercontent.com/4247740/159737775-ac110277-8cc1-46ea-99e1-5ce61395409e.png)